### DISCUSSION TITLE
ABCD release 5.0 site data

### DISCUSSION THREAD

**Discussion by TanFuMiao** [now-i-know-my-abcd/docs#47] [Stats & Code Help] [answered]
I have processed the site data from release 5.0 and have noticed that the site number ranges from 1 to 22. I thought there were only 21 sites according to the ABCD website?

**Comment by kharloews** [comment, marked as answer]
Hi hi! Initially, there was a 22nd site (Mt Sinai) that is no longer part of the study!

**Comment by TanFuMiao** [comment]
Oh I see! So would it still be appropriate to still include those participants in analysis?
